src/editor/components/seSpinInput.js

```diff
@@ -83,6 +84,7 @@ export class SESpinInput extends HTMLElement {
     case 'src':
       this.$img.setAttribute('src', newValue);
       this.$label.remove();
+      this.$div.style.width = '82px';
```

Comment:
I dont think we should hard code the width. We could instead
- put the width as an attribute of the web component
- define a global css variable --se-spin-input-width 
- ...

Reply:
modified